src/components/button.js

```diff
@@ -4,37 +4,63 @@ import propTypes from "prop-types";
 
 import Link from "./link";
 
-const ButtonLink = styled(Link)`
-/* Always use uppercase text */
-text-transform: uppercase;
-font-weight: bold;
-
-/* Remove bootstrap border and radius */
-border: 0px;
-border-radius 0px;
-
-color: var(--MHP-white) !important;
-background-color: var(--MHP-purple);
-
-&:hover {
-  /* Slightly darker purple */
-  background-color: #37279e;
-} 
+const universalButtonStyle = `
+  /* Always use uppercase text */
+  text-transform: uppercase;
+  font-weight: bold;
+
+  /* Remove bootstrap border and radius */
+  border: 0px;
+  border-radius 0px;
+
+  color: var(--MHP-white) !important;
+  background-color: var(--MHP-purple);
+
+  &:hover {
+    /* Slightly darker purple */
+    background-color: #37279e;
+  } 
+`;
+
+const LinkButton = styled(Link)`
+  ${universalButtonStyle}
+`;
+const RealButton = styled.button`
```

Comment:
Would `SubmitButton` be more descriptive?

Reply:
Definitely would be!